tutorials/pub-sub/README.md

```diff
@@ -537,6 +581,57 @@ def c_subscriber():
 
 Note: if `flush=True` is not set, logs will not appear when running `kubectl get logs...`. This is a product of Python's output buffering.
 
+### C# message subscriber
```

Comment:
Do we need the swagger?

Reply:
@davidfowl I don't think so, it came with default template and I didn't remove it, Do you want me to remove it?

Reply:
Yep. Lets make this minimal 😄